fig/backends/workspaceone/serverlog/__init__.py

```diff
@@ -0,0 +1,18 @@
+from logging import INFO, Formatter, getLogger
+from ssl import CERT_NONE, PROTOCOL_TLSv1_2
+from tlssyslog.handlers import TLSSysLogHandler
+from ....config import config
+
+serverlog = getLogger('ws1')
+serverlog.setLevel(INFO)
+
+tlsHandler = TLSSysLogHandler(
+    address=(config.get('workspaceone', 'syslog_host'), int(config.get('workspaceone', 'syslog_port'))),
+    ssl_kwargs={'cert_reqs': CERT_NONE, 'ssl_version': PROTOCOL_TLSv1_2}
```

Comment:
Would be great to make this `CERT_NONE` configurable at some point.

Not topic of this pr, though.